edge-modules/DMReceiver/src/MessageBody.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace DMReceiver
+{
+    using System;
+    using Newtonsoft.Json;
+
+    // NOTE: IF CHANGING ANYTHING IN THIS FILE, UPDATE MESSAGEBODY.CS IN TEMPERATURE FILTER
+    // TODO Put message body in a common lib
+
+    /// <summary>
+    ///Body:
+    ///{
+    ///  “machine”:{
+    ///    “temperature”:,
+    ///    “pressure”:
+    ///  },
+    ///  “ambient”:{
+    ///    “temperature”: , 
+    ///    “humidity”:
+    ///  }
+    ///  “timeCreated”:”UTC iso format”
+    ///}
+    ///Units and types:
+    ///Temperature: double, C
+    ///Humidity: int, %
+    ///Pressure: double, psi
+    /// </summary>
+    public class MessageBody
```

Comment:
This class is not used in the receiver, can it be removed?

Reply:
Yes! I though about using it and forgot it there. :) Thanks! Removing. 

---
In reply to: [218899114](https://github.com/Azure/iotedge/pull/316#discussion_r218899114) [](ancestors = 218899114)

Reply:
👍